packages/web/hooks/ui-config/use-add-concentrated-liquidity-config.ts

```diff
@@ -98,5 +101,33 @@ export function useAddConcentratedLiquidityConfig(
     config.poolId,
   ]);
 
-  return { config, addLiquidity };
+  const increaseLiquidity = useCallback(
+    (positionId: string) => {
+      return new Promise<void>(async (resolve, reject) => {
+        const amount0 = config.baseDepositAmountIn.getAmountPrimitive().amount;
+        const amount1 = config.quoteDepositAmountIn.getAmountPrimitive().amount;
+
+        await account.osmosis.sendAddToConcentratedLiquidityPositionMsg(
+          positionId,
+          amount0,
+          amount1,
+          undefined,
+          undefined,
+          (tx) => {
+            if (tx.code) reject(tx.log);
+            resolve();
+          }
+        );
+
+        try {
```

Comment:
What's going on here? the try-catch block is empty

Reply:
Good catch